proto/eth/v2/beacon_block.proto

```diff
@@ -41,8 +41,17 @@ message BlockSSZResponseV2 {
 }
 
 message BeaconBlockContainerV2 {
-  v1.BeaconBlock phase0Block = 1;
-  BeaconBlockAltair altairBlock = 2;
+  oneof block {
+    v1.BeaconBlock phase0Block = 1;
+    BeaconBlockAltair altairBlock = 2;
+  }
+
+  // 96 byte BLS signature from the validator that produced this block.
+  bytes signature = 3 [(ethereum.eth.ext.ssz_size) = "96"];
+}
+
+message SignedBeaconBlockAltair {
```

Comment:
This message is required for `GetBlockSSZV2` because our fastszz library does not support `oneof` (which is used in the declaration of `BeaconBlockContainerV2` message).